### DISCUSSION TITLE
How to overcome sync error in the codesapce ? Ref image attached.

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#81] [Q&A]
@eculler @nquarder 

![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/38798140/14fd7d6c-b8f7-4205-8f6d-c4ab85d15fee)

**Comment by nquarder** [comment]
Hi @DharaniSureshBabu - you can fix this by selecting the three little dots in the upper right from the `Source Control` menu (where you Commit Changes and Sync), then selecting `Pull, Push` and then selecting `Pull (Rebase)`. I am attaching a screenshot that I hope will help. Thanks @DharaniSureshBabu -NQ

![Screen Shot 2024-04-30 at 8 56 22 PM](https://github.com/cu-esiil-edu/2024-data-short-course/assets/16786791/9a641af4-f88a-4c7a-adee-bf82b86d89a1)
